packages/trader/src/Stores/Modules/Trading/Actions/duration.ts

```diff
@@ -46,10 +46,10 @@ const assertDuration = ({
 }: TAssertDurationParams = {}) => {
     const [min, max] = getDurationMinMaxValues(duration_min_max ?? {}, contract_expiry_type ?? '', duration_unit ?? '');
 
-    if (duration && min && duration < min) {
+    if (Number(duration) < Number(min)) {
```

Comment:
L49 and L52 fix a bug introduced during TS migration: 0 duration should be true here